gcc.symbols.imp

```diff
@@ -176,15 +176,6 @@
   { "symbol": [ "std::char_traits", "private", "<ostream>", "public"] },
   { "symbol": [ "std::char_traits", "private", "<istream>", "public"] },
 
-  # std::ptrdiff_t is often an architecture specific definition, force the
```

Comment:
Please leave this comment, here and in `iwyu_include_picker.cc`

Reply:
Ok.

Reply:
I've moved it to a separate commit.  Is that ok?